app/components/Application/ApplicationDecision.tsx

```diff
@@ -0,0 +1,65 @@
+import React from 'react';
+import {Alert} from 'react-bootstrap';
+
+interface Props {
+  actionRequired: Boolean;
+  decision: string;
+  reviewComments: string[];
+}
+
+const BS_ALERT = {
+  REQUESTED_CHANGES: 'warning',
+  REJECTED: 'danger',
+  APPROVED: 'success'
+};
+
+const DECISION_TEXT = {
+  REQUESTED_CHANGES: '',
+  REJECTED:
+    'Thank you for applying to the CIIP. Unfortunately this submission did not meet program requirements and a payment cannot be granted.',
+  APPROVED:
+    'This application has been approved. Your incentive payment is being processed. No further action is required from you at this time.'
+};
+
+const DECISION_PRETTY = {
+  REQUESTED_CHANGES: 'Changes Requested',
+  REJECTED: 'Rejected',
+  APPROVED: 'Approved'
+};
+
+const ApplicationDecision: React.FunctionComponent<Props> = ({
+  actionRequired,
+  decision,
+  reviewComments,
+  children
+}) => {
+  const heading = actionRequired
+    ? 'Action Required'
+    : decision === 'REQUESTED_CHANGES'
+    ? 'Changes Requested'
+    : `Decision: ${DECISION_PRETTY[decision]}`;
```

Comment:
Not a big fan of double ternary operators, can you use if/else instead?

Reply:
I guess I gravitated that way because the if/else if/else alternative requires re-assignment so `heading` can't be a constant, which it should be ideally. 🤔 

I broke it into separate lines for readability. I'm not sure if the below is actually more readable?

```
  let heading;
  if (actionRequired) {
    heading = 'Action Required';
  } else if (decision === 'REQUESTED_CHANGES') {
    heading = getUserFriendlyStatusLabel(decision);
  } else {
    heading = `Decision: ${getUserFriendlyStatusLabel(decision)}`;
  }
```

Reply:
It's a bit more verbose, but I think that the order of operations is more explicit